### ISSUE TITLE
oelint.vars.dependsordered FP when remove is used

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#325]
```
DEPENDS = "abc def uuu"
DEPENDS:remove = "def"
```
gives a false positive for the second line
